test/config/createremovetargettest.go

```diff
@@ -64,7 +85,9 @@ func (s *TestSuite) TestCreatedRemovedTarget(t *testing.T) {
 	// Set a value using gNMI client - target is down
 	setPath2 := gnmiutils.GetTargetPathWithValue(createRemoveTargetModTargetName, createRemoveTargetModPath, createRemoveTargetModValue2, proto.StringVal)
 
-	_, _ = gnmiutils.SetGNMIValueOrFail(ctx, t, c, setPath2, gnmiutils.NoPaths, gnmiutils.NoExtensions)
+	setReq.UpdatePaths = setPath2
+	setReq.Extensions = nil
```

Comment:
I think the default should be nil so you don't need to set it to nil.

Reply:
setReq is reused, so the value is overwritten